R/colour_aliases_1in1000.R

```diff
@@ -0,0 +1,16 @@
+#' Named colour vector associated with 1 in 1000
```

Comment:
I'm going to disregard this suggestion since, as Alex indicated below, the named vectors are required and putting them into a dataframe wouldn't make much sense.